sysconfigs/daint/packages.yaml

```diff
@@ -1,45 +1,16 @@
 packages:
   all:
-      # default compilers defined by the system
-      # these reflect the current installed PE
-      compiler: [gcc@10.2.0, gcc@9.3.0, gcc@8.3.0, pgi@20.1.1, pgi@20.1.0, pgi@21.3.0, nvhpc@21.3.0, cce@12.0.2, cce@11.0.0, intel@19.0.1.144]
-      providers:
-          mpi: [mpich]
-          # if the mpich package support +cuda in the future it needs to be put there
-          mpicuda: [mpich]
-          mkl: [intel-mkl]
-          blas: [cray-libsci_acc, cray-libsci, intel-mkl]
-          scalapack: [cray-libsci_acc, cray-libsci, intel-mkl]
-          pkgconfig: [pkg-config]
-      target: [haswell]
```

Comment:
The `target: [haswell]` was removed. Why?

Reply:
Let me check